packages/circle-react-elements/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,50 @@
+import { ArrowUpRight, CheckCircle } from 'lucide-react';
+
+import { Button } from '~/components/ui/button';
+
+export interface NotificationProps {
+  /** The wallet */
+  onClose?: () => void;
+  description: string | React.ReactElement;
+  externalLink?: string;
+  title: string;
+}
+
+/**
+ * Helpers for obtaining a wallet's on-chain address:
+ * a QR code that encodes the address and elements for viewing the address and copying it to the clipboard
+ */
+export function Notification({
+  onClose,
+  title,
+  description,
+  externalLink,
+}: NotificationProps) {
+  return (
+    <div className="flex flex-col items-center">
+      <div className="rounded-full bg-green-200 p-1 border-green-100 border-8">
+        <CheckCircle className="text-green-600" />
```

Comment:
The green checkbox in your implementation looks quite different from the design. Could you adjust it to match the design more closely?

in this PR:
<img width="96" alt="Screenshot 2025-01-28 at 11 05 57" src="https://github.com/user-attachments/assets/6cb4906c-ae40-4f9c-978c-cd25eb6cf994" />
<img width="116" alt="Screenshot 2025-01-28 at 11 06 01" src="https://github.com/user-attachments/assets/d4e2584a-bcf3-4675-97d8-22618fd8be94" />

in the design:
<img width="74" alt="Screenshot 2025-01-28 at 11 06 08" src="https://github.com/user-attachments/assets/d1523a37-412a-44cd-af2f-54a9fe651c26" />